src/gui/CMakeLists.txt

```diff
@@ -231,13 +229,13 @@ if(NOT BUILD_OWNCLOUD_OSX_BUNDLE)
     # we may not add MACOSX_BUNDLE here, if not building one
```

Comment:
Building a bundle is the only supported option for a long long time, maybe we should get rid of `BUILD_OWNCLOUD_OSX_BUNDLE` and simplify this code a bit?
And also adjust comments?

Reply:
I think I'll have to rework some stuff anyhow..... ( mac is currently broken on this branch)